src/main/java/uk/gov/hmcts/reform/et/syaapi/controllers/ManageCaseController.java

```diff
@@ -40,10 +42,12 @@
 @RequiredArgsConstructor
 @RestController
 @RequestMapping("/cases")
+@SuppressWarnings({"PMD.TooManyMethods"})
```

Comment:
Noticed that you were able to add a PMD warning without modifying the number at the bottom of build.gradle. Please review to the build.gradle so the number of allowed suppressions is reduced